### DISCUSSION TITLE
Which forms of documentation have you found most useful in the past?

### DISCUSSION THREAD

**Discussion by shonfeder** [informalsystems/quint#404] [Q&A]
In the long term, we'd like all the best forms of documentation covering all the quadrants of the [divio documentation system](https://documentation.divio.com/) and beyond: we'd like integrated interactive tutorials and online-sandboxes and a GPT powered Quint-clippy and a text book and blog posts and tons of Stack Overflow questions answered etc. :infinity: :smile: 

But in the short and **immediate** term, we should focus our documentation efforts on what is most needed and what is most useful to our internal @informalsystems/tnt-co-designers 

If you are going to be writing Quint specs in the next 6-12 weeks, and you have some insight into the kinds of documentation you usually find most useful, please reply here giving us insight into your needs and preferences!

Please also note that we will be dedicating time for one-on-one and group sessions in addition, and we will of course be available to answer any questions as they arise :)

**Comment by mpoke** [comment]
Thanks @shonfeder for this initiative. IMO, the best way to start using TNT would be to have a simple example (e.g., token transfer) documented: what the code is doing, how to use the code, how to debug. Then some iteration on how to add extra features, such as interfaces, different data structures.

**Reply by shonfeder** [reply]
This is a great idea, thanks: https://github.com/informalsystems/quint/issues/489

**Comment by hvanz** [comment]
What about a TNT vs TLA+ comparison table? I think this would be quite useful for people that already know TLA+.

**Reply by shonfeder** [reply]
Thanks so much for the suggestion, tracked in: https://github.com/informalsystems/quint/issues/488

**Reply by konnov** [reply]
I think it is currently addressed via https://github.com/informalsystems/quint/blob/main/doc/quint-cheatsheet.pdf
